WordPress/Classes/ViewRelated/QR Login/Coordinators/QRLoginScanningCoordinator.swift

```diff
@@ -9,12 +10,30 @@ class QRLoginScanningCoordinator: NSObject {
     let parentCoordinator: QRLoginCoordinator
     let view: QRLoginScanningView
 
+    // Camera Handling
+    var session: AVCaptureSession?
+    var cameraDevice: AVCaptureDevice?
+
     init(view: QRLoginScanningView, parentCoordinator: QRLoginCoordinator) {
         self.view = view
         self.parentCoordinator = parentCoordinator
     }
+
+    func start() {
+        configureCamera()
+
+        guard cameraDevice == nil else {
+            configureCameraPreview()
+            return
+        }
+
+        showNoCameraError()
```

Comment:
[nit] I think this is more readable, wdyt?
```suggestion
        guard cameraDevice != nil else {
            showNoCameraError()
            return
        }

        configureCameraPreview()
```

Reply:
I agree, updated here: b53474c47bbde8e50d149b604801fc19b79881ea